agent_factory/scaffold/orchestrator.py

```
@@ -0,0 +1,362 @@
+            "tasks_queued": len(state.tasks_queued),
+            "tasks_in_progress": len(state.tasks_in_progress),
+            "tasks_completed": len(state.tasks_completed),
+            "tasks_failed": len(state.tasks_failed),
+            "total_cost": state.total_cost,
+            "total_duration_sec": state.total_duration_sec
+        }
+
+        # Add safety stats if available
+        try:
+            summary["safety_stats"] = self.session_mgr.safety.get_stats()
+        except:
```

Ruff (0.14.8)

Do not use bare `except`

(E722)